src/libDirectoryService/Coinbase.cpp

```diff
@@ -371,6 +376,7 @@ void DirectoryService::InitCoinbase() {
           if (GUARD_MODE) {
             if (Guard::GetInstance().IsNodeInDSGuardList(pk) ||
                 Guard::GetInstance().IsNodeInShardGuardList(pk)) {
+              suc_counter++;
```

Comment:
`suc_counter` means success counter?  If so, why do we increment it here if we don't actually credit `reward_each`?

Reply:
because we want to divide the total rewards with the whole size of the network, and skip those guard node when do the  actual rewarding.